AssertionError with semanticdb-scalac

While compiling Ammonite with semanticdb 3.2 I hit on a large number of

```
[warn] failed to generate semanticdb for /Users/ollie/dev/Ammonite/amm/interp/src/main/scala/ammonite/interp/Compiler.scala:
[warn] java.lang.AssertionError: assertion failed
[warn] 	at scala.Predef$.assert(Predef.scala:204)
[warn] 	at scala.meta.internal.semanticdb.scalac.DocumentOps$XtensionCompilationUnitDocument$traverser$4$.$anonfun$tryFindMtree$4(DocumentOps.scala:209)
```

--- a/semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/DocumentOps.scala
+++ b/semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/DocumentOps.scala
@@ -206,8 +206,9 @@ trait DocumentOps { self: DatabaseOps =>
                   (todoTpe1 ++ todoTpe2).foreach { tgs =>
                     if (tgs.isSemanticdbLocal) {
                       val tms = tgs.toSemantic
-                      assert(tms != m.Symbol.None)
-                      if (!denotations.contains(tms)) add(tms, tgs)
+                      if (tms != m.Symbol.None && !denotations.contains(tms)) {
+                        add(tms, tgs)
+                      }
                     }
                   }
                 }

